CHANGELOG.md

```diff
@@ -1,3 +1,108 @@
+# 0.0.106 - 2022-03-30
+
+## API Updates
+ * Minimum supported rust version (MSRV) is now 1.41.1 (#1310).
+ * `PhantomRouteHints`, `FixedPenaltyScorer`, and `ScoringParameters` now
+   implement `Clone` (#1346).
+ * `DefaultRouter` and `find_route` take an additional random seed to improve
+   privacy by adding a random CLTV expiry offset to each path's final hop. This
+   helps obscure the intended recipient from adversarial intermediate hops
+   (#1286). The seed is  also used to randomize candidate paths during route
+   selection (#1359).
+ * `PeerManager::timer_tick_occurred` will disconnect an unresponsive peer after
+   four ticks, corresponding to 40 seconds with the new recommendation to call
+   the method every 10 seconds (#1348).
+ * The `lightning-block-sync` crate's `init::synchronize_listeners` method
+   interface has been relaxed to support multithreaded environments (#1349).
+ * `ChannelManager::create_inbound_payment_for_hash`'s documentation has been
+   corrected to remove the one-year restriction on `invoice_expiry_delta_secs`,
+   which is only applicable to the deprecated `create_inbound_payment_legacy`
+   and `create_inbound_payment_for_hash_legacy` methods (#1341).
```

Comment:
IMHO, it's better to announce a timeline or release number in the deprecation warning, instead to say "will be removed soon" which opens the way for interpretation and thus user bargaining. I know, for a maintainer it might be preferable to use "soon" as it give more room to adapt in function of the user base, though let's be aware it's a double-edge (Yeah the project is young, we might not have a consistent deprecation process that early).

Reply:
True, though note this isn't related to any changes for this release.